packages/paste-core/utilities/box/src/SafelySpreadProps.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  LAYOUT_PROPS,
+  SPACE_PROPS,
+  BACKGROUND_PROPS,
+  BORDER_PROPS,
+  SHADOW_PROPS,
+  POSITION_PROPS,
+  FLEXBOX_PROPS,
+} from '@twilio-paste/style-props';
+
+export const BOX_PROPS_TO_BLOCK = [
+  ...LAYOUT_PROPS,
+  ...SPACE_PROPS,
+  ...BACKGROUND_PROPS,
+  ...BORDER_PROPS,
+  ...SHADOW_PROPS,
+  ...POSITION_PROPS,
+  ...FLEXBOX_PROPS,
+  'backgroundColor',
+  'borderColor',
+  'className',
+  'style',
+];
+
+export const safelySpreadBoxProps = (props: {}, compName: string): void => {
```

Comment:
I'm not a fan of the compName argument here because it complicates the API. When we throw, the stacktrace should show with file the error originated from.